internal/validate/validate.go

```diff
@@ -40,6 +40,25 @@ func CheckEthereumAddress(parameter interface{}, variableName string) error {
 	return nil
 }
 
+func CheckEthereumAddressList(parameter interface{}, variableName string) error {
```

Comment:
This is a copy of `CheckEthereumAddress`

Reply:
It is not a copy since it is working for the list of eth addresses

Reply:
Tests added